app/dao/notifications_dao.py

```diff
@@ -815,12 +815,16 @@ def letters_missing_from_sending_bucket(seconds_to_subtract):
     return notifications
 
 
-def dao_precompiled_letters_still_pending_virus_check():
-    ten_minutes_ago = datetime.utcnow() - timedelta(seconds=600)
+def dao_precompiled_letters_still_pending_virus_check(max_minutes_ago_to_check):
+    thirty_minutes_ago = datetime.utcnow() - timedelta(minutes=max_minutes_ago_to_check)
```

Comment:
Can we not call this `thirty_minutes_ago` if it's actually just `max_minutes_ago_to_check`? ;)

Reply:
oops yes we definitely can, good spot

Reply:
Squashed